stable/openstack-base/README.md

```diff
@@ -1,333 +1,242 @@
-# Basic OpenStack cloud
+# Basic OpenStack Cloud
 
-The `openstack-base` bundle currently deploys a basic OpenStack cloud with
-these foundational elements:
-
-- Ubuntu 18.04 LTS (Bionic)
-- OpenStack Train
-- Ceph Mimic
+*DEV/TEST ONLY*: This unstable, development example bundle deploys a basic OpenStack Cloud (Ussuri with Ceph Octopus) on Ubuntu 20.04 LTS (Focal), providing Dashboard, Compute, Network, Block Storage, Object Storage, Identity and Image services.  See also: [Stable Bundles](https://jujucharms.com/u/openstack-charmers).
```

Comment:
We are in the `stable/` folder so it's kind of strange to say that this is "unstable". Is it intended? Is it because this is a bit more stable than the other folders, yet not as stable as https://jujucharms.com/u/openstack-charmers ?

Reply:
Ah, thanks, it was supposed to be changed in the next commit. The idea of the workflow is to copy whatever is in the development folders verbatim in one commit and promote to stable in the final commit. Thank you for catching this and I'll sort it momentarilly.